nucypher/blockchain/eth/clients.py

```diff
@@ -288,6 +303,13 @@ def unlock_account(self, address, password):
     def sync(self, *args, **kwargs):
         return True
 
+    def get_signing_key(self, account):
```

Comment:
does Ganache not support the `signTransaction` and `sendRawTransaction` endpoints?

Reply:
@KPrasch apparently not... :|

https://github.com/trufflesuite/ganache-core/issues/408

Reply:
OK!  We need to handle the local private key then, much like `EthereumTesterClient`; However, let's remove all the private key handling from the base class, since we only want to do so in-tests.